dashboard/src/components/AppView/AccessURLTable/AccessURLTable.tsx

```diff
@@ -6,44 +6,48 @@ import { GetURLItemFromIngress } from "./AccessURLItem/AccessURLIngressHelper";
 import { GetURLItemFromService } from "./AccessURLItem/AccessURLServiceHelper";
 
 interface IServiceTableProps {
-  services: { [s: string]: IResource };
-  ingresses: { [i: string]: IResource };
+  services: IResource[];
```

Comment:
Could we instead of using IResource use something more specific in reference to the Service? I am not sure if we have the types for that though.

Reply:
we don't have a type `IService` but it's not needed since it follows the generic layout (metadata, spec and status keys).